### ISSUE TITLE
Feature Request: Redirecting to url even when logged in

### ISSUE THREAD

**Issue by rarog** [LM-Commons/LmcUser#24]
Current behaviour is that the redirection is done on login. Let's say I have n tabs, where I'm logged in, if I open all n of them after logout, I get a login screen with redirection parameter. If I log in in one of the tabs, I'll be redirected to the url in the parameter. All the other n-1 tabs will be redirected to the default url and the original url in the redirection parameter is "lost".

If this is interesting and useful to other people, I could start working on it.

**Comment by visto9259** [comment]
@rarog I know this is way late to respond to this.

Is this still something that is requested?  Did you work on a solution of you own?

**Comment by rarog** [comment]
@visto9259 That's still something of interest and improvement of the behavior.
I never started working on it and shelved the idea for later and actually forgot about this.

**Comment by visto9259** [comment]
@rarog Where do you set the redirect param? In a guard or an event listener that checks if the user is logged in? Or in a controller that redirects when the user is not logged in?

**Comment by rarog** [comment]
The controller AFAIR, which "stores" the url as GET parameter in the url. use_redirect_parameter_if_present setting is it.

**Comment by visto9259** [comment]
Let me just make sure I understand the flow of your use case. 
You have n tabs opened, all logged in. You logged out in one of them. You reopen all n tabs and you app redirect all n tabs to login with a redirect to the original page. 

You log back in in one tab which gets redirected to the original page. 

On the other tabs, what action do you do? You log in again? 

The login action will redirect to the default route defined in the options if the user is already logged in. It ignores the redirect parameter. This is probably the issue you have. 

It should be an easy fix to use the redirect parameter if set and if configured to be used.

**Comment by rarog** [comment]
Yes, that's the exact description of the scenario for the issue.
